app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -441,11 +442,12 @@ public function getAssetBySerial(Request $request)
      * @since [v3.0]
      * @return Redirect
      */
-    public function getAssetByTag(Request $request)
+    public function getAssetByTag($tag=null, Request $request)
```

Comment:
TINY nit here (and I’d accept this without this change) but stylistically, we typically order method parameters with `Request` first, required (not nullable) variables next, and nullable/optional variables last.

Reply:
in case its useful, i swapped these parameters on my fork since it was a relatively easy, low risk change to make

Reply:
Yep. This was a derp on my part. Sorry.